mlrun/config.py

```diff
@@ -120,6 +120,19 @@
     "ignored_notebook_tags": "",
     # when set it will force the local=True in run_function(), set to "auto" will run local if there is no k8s
     "force_run_local": "auto",
+    # timeout to wait for background task to be updated / finished by the worker responsible for the task
```

Comment:
This comment is not in place
In addition, specify the timeout units (seconds I guess ?)